test/suite/test_backup04.py

```diff
@@ -118,8 +94,9 @@ def test_incremental_backup(self):
         self.populate_with_string(self.uri, self.dsize, self.nops)
```

Comment:
This function would be better named `test_log_incremental_backup`.

This test was written before block-based incremental backup existed. The comments in this file (and any others using log-based incremental) should be updated to explicitly say log-based incremental backup.

Reply:
done